tensorflow_addons/custom_ops/activations/cc/kernels/hardshrink_op.h

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 #define TENSORFLOW_ADDONS_ACTIVATIONS_KERNELS_HARDSHRINK_OP_H_
 
 #define EIGEN_USE_THREADS
+#define _USE_MATH_DEFINES
+#include <cmath>
```

Comment:
Just to confirm if this is for windows build? Thanks!

Reply:
Yes. It fails to compile otherwise for a core lib missing the include (log in original post but it's a bit verbose). It didn't happen on the RC, only nightly which is strange.

I can mark an issue to investigate and remove these later if we can get a fix on tf core

Reply:
Also working on windows CI sho it'll be easier to test and review these windows issues

Reply:
Seems that core TF does the same thing recently.

https://github.com/tensorflow/tensorflow/search?q=_USE_MATH_DEFINES&type=Commits